setup/docker/build.sh

```diff
@@ -13,12 +13,13 @@ tmp="./setup/docker/tmp"
 
 mkdir -p ${tmp}
 
-cat ./setup/unix/traccar.xml | awk '/web.path/ && !modif { printf("    <entry key=\"web.debug\">true</entry>\n"); modif=1 } {print}' > ${tmp}/traccar.xml
+cat ./setup/traccar.xml | awk '/web.path/ && !modif { printf("    <entry key=\"web.debug\">true</entry>\n"); modif=1 } {print}' > ${tmp}/traccar.xml
```

Comment:
I was about to make a PR for the same thing when I saw yours.

However, I think in order to properly set _web.debug=true_, you should operate on `./setup/default.xml` instead because you won't be able to find `/web.path/` on `traccar.xml`.